packages/nimble-components/src/label-provider/core/label-tokens.ts

```diff
@@ -0,0 +1,17 @@
+import { DesignToken } from '@microsoft/fast-foundation';
+import { coreLabelDefaults } from './label-token-defaults';
+
+export const alertDismissLabel = DesignToken.create<string>({
```

Comment:
This label/token might apply to more components in the future - the [HLD had a comment](https://github.com/ni/nimble/pull/1272#discussion_r1228408606) with `banner-dismiss` and `toast-dismiss` as the example (but we don't have toast yet).

So I tried to pick a more general name and ended up with `alert-dismiss`. If people thinks it makes more sense to go with `banner-dismiss` (until/unless we have a 2nd component being added that would use it) let me know.

Reply:
Maybe popup-dismiss? We have popupBorderColor in our existing tokens so popup as a generic element is there, don't see one for alert (which feels a bit too specific? like a snackbar or even a persistent tooltip etc feel like they could have a dismiss button)

Reply:
Updated